packages/llamaindex/src/tools/AzureDynamicSessionTool.ts

```diff
@@ -0,0 +1,355 @@
+import {
+  DefaultAzureCredential,
+  getBearerTokenProvider,
+} from "@azure/identity";
+import { getEnv } from "@llamaindex/env";
+import crypto from "node:crypto";
+import { createWriteStream } from "node:fs";
+import { readFile } from "node:fs/promises";
+import { dirname, join, resolve } from "node:path";
+import { Readable } from "node:stream";
+import { fileURLToPath } from "node:url";
```

Comment:
if this is only for node.js, rename to AzureDynamicSessionTool.node.ts and only export it in src/index.ts

Reply:
done.